ocaml/src/osd_access.ml

```diff
@@ -158,8 +158,18 @@ module Osd_pool = struct
               let open Lwt_pool2 in
               match exn with
               | Asd_protocol.Protocol.Error.Exn _ -> Keep
+              | Unix.Unix_error(Unix.EFAULT, _, _) ->
+                 Lwt_log.ign_info_f
+                   ~exn
```

Comment:
drop `~exn` (it doesn't really work and you put the callstack in the message anyway)